transport/tchannel/outbound.go

```diff
@@ -64,6 +64,17 @@ type outbound struct {
 	HostPort string
 }
 
+func (o outbound) Start() error {
+	// TODO: Should we create the connection to HostPort (if specified) here or
```

Comment:
FWIW, Simon mentioned we'll want to solve for warmup as well.

Sent from my iPhone

> On Jun 23, 2016, at 8:15 AM, Prashant Varanasi <notifications@github.com> wrote:
> 
> In transport/tchannel/outbound.go:
> 
> > @@ -64,6 +64,17 @@ type outbound struct {
> >  	HostPort string
> >  }
> >  
> > +func (o outbound) Start() error {
> > +	// TODO: Should we create the connection to HostPort (if specified) here or
> Hmm, I imagine if an application specifies an outbound, it will most likely use it, so connecting earlier might be better. However, I'd also be fine with a "warmup" phase in services that makes a bunch of requests to warm up all the connections etc, before it's officially up and ready to serve traffic.
> 
> ―
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>